tests/test_runtime.py

```diff
@@ -0,0 +1,736 @@
+import os
+
+import numpy as np
+import pennylane as qml
+import pytest
+
+from qiskit import IBMQ
+
+from pennylane_qiskit import IBMQCircuitRunnerDevice
+from pennylane_qiskit import IBMQSamplerDevice
+from pennylane_qiskit.vqe.vqe_runner import vqe_runner, upload_vqe_runner, delete_vqe_runner
+
+
+@pytest.fixture
+def token():
```

Comment:
This function should be moved to `conftest.py` from `tests/test_ibmq.py` and used in this file as a fixture. Note: line 18 was recently updated.